README.md

```diff
@@ -4,12 +4,29 @@ This tutorial demonstrates the use of MicroProfile technologies for implementing
 made up of a number of Open Liberty Guides each of which demonstrates different MicroProfile technologies. Each Guide is 
 designed to be taken independently so if you just want to learn about a specific technology you can just take that guide. If, however, you're goal is to learn about all the MicroProfile capabilities, then working through them in the order shown below is recommended.
 
+## Tutorial Preparation
+
+The save time during the tutorial, it's best to set up your machine beforehand. The instructions below show the pre-requisites to install and how to avoid lengthy downloads.
```

Comment:
To* save time. I am now just being picky!

Reply:
Don't mind picky.  Thanks.